src/Symfony/Component/Clock/Clock.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A global clock.
+ *
+ * Defined as enum to be final and non-instantiable.
+ */
+enum Clock
+{
+    public static function now(): \DateTimeImmutable
+    {
+        return self::clock()->now();
+    }
+
+    public static function clock(ClockInterface $newClock = null): ClockInterface
```

Comment:
perhaps expect a PSR clock here (same for ClockAwareTrait btw)

Reply:
no need: as soon as one uses this class, they couple to the component, so no need for the PSR interface